paddle/fluid/operators/sequence_ops/sequence_pool_op.cc

```diff
@@ -37,6 +37,7 @@ class SequencePoolOp : public framework::OperatorWithKernel {
   }
 };
 
+template <typename T>
 class SequencePoolOpMaker : public framework::OpProtoAndCheckerMaker {
```

Comment:
Please don't use `template ` here, it is an error usage.

Reply:
DONE